src/m365/planner/commands/task/task-reference-remove.ts

```diff
@@ -0,0 +1,155 @@
+import * as _ from 'lodash';
+import { Cli, Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  alias?: string;
+  taskId: string;
```

Comment:
usually when we have param1 or param2 kind of approach we make each of those nullable 🤔
I suppose we should add `url?` here

Reply:
both url and taskId should be nullable